packages/web-components/src/components/dotcom-shell/__stories__/dotcom-shell.stories.ts

```diff
@@ -691,26 +837,23 @@ export default {
     useRawContainer: true,
     knobs: {
       escapeHTML: false,
-      MastheadComposite: ({ groupId }) => ({
+      DotcomShell: ({ groupId }) => ({
         platform: select('Platform (platform)', { none: null, platform: platformData.name }, null, groupId),
-        hasProfile: boolean('show the profile functionality (profile)', true, groupId),
-        hasSearch: boolean('show the search functionality (search)', true, groupId),
-        searchPlaceholder: textNullable('search placeholder (searchPlaceholder)', inPercy() ? ' ' : 'Search all of IBM', groupId),
-        selectedMenuItem: textNullable('selected menu item (selected-menu-item)', 'Services & Consulting', groupId),
-        userStatus: select('The user authenticated status (user-status)', userStatuses, userStatuses.unauthenticated, groupId),
-      }),
-      FooterComposite: ({ groupId }) => ({
-        disableLocaleButton: boolean('hide the locale button (disable-locale-button)', false, groupId),
+        hasProfile: boolean('Show profile in masthead (profile)', true, groupId),
+        hasSearch: boolean('Show search in masthead (search)', true, groupId),
+        searchPlaceholder: textNullable('Search placeholder (searchPlaceholder)', 'Search all of IBM', groupId),
+        selectedMenuItem: textNullable('Selected menu item (selected-menu-item)', 'Services & Consulting', groupId),
```

Comment:
This should be a `select()`, per [Shixie's comment](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/pull/5565#issuecomment-811201753) – can just create a simple array with the existing menu items, maybe?